src/cfml/system/modules_app/system-commands/commands/browse.cfc

```diff
@@ -12,9 +12,9 @@ component {
 	/**
 	 * @URI.hint The URI to open as you would type it into your browser's address bar
  	 **/
-	function run( required URI )  {
+	function run( required URI, BROWSER )  {
```

Comment:
Again `browser` would not be upper case here. URl is only uppercase because it is an acronym.